server/lib/update-linked-fields.js

```diff
@@ -1,54 +1,62 @@
 import Customer from '../models/customer'
 import Volunteer from '../models/volunteer'
 import Donor from '../models/donor'
+import {clientRoles} from '../../common/constants'
 
 /**
-* Update Volunteer fields of address
+* Update fields of address
 */
-export async function updateVolunteer(reqbodyfields, user_id) {
+ async function update(Model, reqbodyfields, user_id) {
```

Comment:
use camel case - `reqBodyFields, userId`

i'd call it updatedFields instead too